downloads/downloads-api/src/main/java/com/duckduckgo/downloads/api/DownloadStateListener.kt

```diff
@@ -25,3 +29,17 @@ interface DownloadStateListener {
      */
     fun commands(): Flow<DownloadCommand>
 }
+
+interface DownloadConfirmationDialogListener {
+    fun continueDownload(pendingFileDownload: PendingFileDownload)
+    fun cancelDownload()
+}
+
+interface DownloadConfirmation {
+    fun instance(pendingDownload: PendingFileDownload): BottomSheetDialogFragment
```

Comment:
Would’ve preferred to have this to be showConfirmation(PendingFileDownload, tag) but I saw that there’s much more logic in the Browsertabfragment usage. So this is OK for now.

Reply:
Yeah overall is more like a temporary kinda thing because ideally I'd love to have something similar to GlobalActivityStarter but for Fragments.